account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -109,6 +109,12 @@ def search_entries(self):
         context = self.env.context.copy()
         context['line_ids'] = self.filter_lines(lines)
         context['populate_results'] = self.populate_results
+        if payment.payment_order_type == 'payment':
```

Comment:
I'd say it's a matter of taste. I personally find @adrienpeiffer 's version a tad easier to read.